Add sub page

MaciejSurowiec opened the pull request:
just to test how it looks

MaciejSurowiec committed:
Add sub page

MaciejSurowiec commented:
/deploy

MaciejSurowiec committed:
Add Html subpages

MaciejSurowiec committed:
add new buttons

MaciejSurowiec committed:
add css

szwedosz commented on BeerDiplomaWebpage/src/app/auth/auth.component.ts:
```diff
@@ -0,0 +1,27 @@
+export class AuthComponent implements OnInit {
+  authType: String = '';
+  title: String = '';
+  jebanyan: Boolean = true;
```
what is this? xD

MaciejSurowiec replied:
It’s just testing value, and it supposed to be named „jebanyangular” xd

szwedosz commented on BeerDiplomaWebpage/src/app/home/home.component.html:
```diff
@@ -0,0 +1,3 @@
+<h1 class="center">Strona Glowna</h1>
```
główna

szwedosz commented on BeerDiplomaWebpage/src/styles.css:
```diff
@@ -1 +1,61 @@
+border-style: none;
+}
+
+.form{
```
style - you are missing " " before { everywhere

szwedosz commented on BeerDiplomaWebpage/src/styles.css:
```diff
@@ -1 +1,61 @@
-/* You can add global styles to this file, and also import other style files */
+.form-group{
+border-style: none;
```
incorret indent

MaciejSurowiec committed:
add code refactore

ignore commented parts, it's just logic for another pr. now I want to see how this looks on prod
